### ISSUE TITLE
Dailies not resetting after tavern rest

### ISSUE THREAD

**Issue by dmlodzik** [HabitRPG/habitica#1102]
I set myself to on vacation mode through the tavern for 4 days while I was out of town.  I came back Monday, and took myself out of vacation mode.  It is now thursday, and my Dailies have not rest, all of them are greyed out and have been that way since I returned.
![screen shot 2013-05-30 at 7 11 35 pm](https://f.cloud.github.com/assets/4575347/588507/b5b85f08-c986-11e2-8949-a4fb5712652e.jpg)

**Comment by lefnire** [comment]
any javascript errors?

**Comment by zazzerpaan** [comment]
Hello, sorry, I'm not too sure of the protocol but I hope it's ok to jump in because I've had this too. I had to go into the inn while my computer was being repaired two weeks ago, and the reset of my dailies and habits has been very patchy since then. My dailies haven't reset for the last couple of days, and while they sort of reset last night, streaks I should have broken have remained static. Single action dailies - one with just a plus or minus, I'm so sorry, I can't remember the name for them - didn't reset last night. I've rechecked in and out of the inn to make sure I'm properly out of it, I've changed my start of day in case that was the issue, but none of this seems to have worked.

I don't use HabitRPG on multiple devices, or even multiple browsers. Dailies are a huge part of what makes HabitRPG an invaluable resource for me, so I hope this information can help. I use windows 7 and google chrome version 27.0.1453.110, and there weren't any javascript errors in the console.
